pypeit/specobj.py

```diff
@@ -218,7 +218,7 @@ def from_arrays(cls, PYPELINE:str, wave:np.ndarray, counts:np.ndarray, ivar:np.n
         for item, attr in zip([wave, counts, ivar], ['_WAVE', '_COUNTS', '_COUNTS_IVAR']):
             setattr(slf, mode+attr, item.astype(float))
         # Mask
-        slf[mode+'_MASK'] = slf[mode+'_COUNTS_IVAR'] > 0.
+        slf[mode+'_MASK'] = slf[mode+'_COUNTS'] > 0.
```

Comment:
I get this for fluxing, but if this MASK is used elsewhere then 
I think it should be on `IVAR`.

Reply:
I changed that because some of the throughput fits files had 0 values for the counts, which would cause the `_COUNTS_IVAR` to have a +infinity due to the divide by 0. This also happens when I padded out the length of an odd `COUNTS` array. 
As an alternative I changed it back to use `_COUNTS_IVAR`, but added a check for `isfinite` for the mask.